lib/verdict/experiment.rb

```diff
@@ -193,6 +176,10 @@ def everybody_qualifies?
     !has_qualifier?
   end
 
+  def subject_identifier(subject)
```

Comment:
let's get rid of this method, and instead use the existing `retrieve_subject_identifier`